### ISSUE TITLE
Disappearing unit tests

### ISSUE THREAD

**Issue by jdavies-st** [spacetelescope/jwst#2789]
Our unit tests seem to be disappearing.

We used to have 1370 unit tests.  Now we only have 913.  Time machine:
```
(jwst_dev) rey> git status
On branch master
Your branch is up to date with 'jwst/master'.

(jwst_dev) rey> git checkout HEAD^ ; pytest --collect-only | grep collected
HEAD is now at f0f22e4d... Merge pull request #2781 from nden/add-new-nrs-types
collected 913 items / 1 errors

(jwst_dev) rey> git checkout HEAD^ ; pytest --collect-only | grep collected
Previous HEAD position was f0f22e4d... Merge pull request #2781 from nden/add-new-nrs-types
HEAD is now at c50c9cfe... Association cumulative patch 0.12.1 (#2780)
collected 913 items / 1 errors

(jwst_dev) rey> git checkout HEAD^ ; pytest --collect-only | grep collected
Previous HEAD position was c50c9cfe... Association cumulative patch 0.12.1 (#2780)
HEAD is now at 58943df4... Merge pull request #2779 from sosey/docs
collected 911 items / 1 errors

(jwst_dev) rey> git checkout HEAD^ ; pytest --collect-only | grep collected
Previous HEAD position was 58943df4... Merge pull request #2779 from sosey/docs
HEAD is now at 65e58aa4... Ensure coronagraphic associations input/output integrations (#2773)
collected 911 items / 1 errors

(jwst_dev) rey> git checkout HEAD^ ; pytest --collect-only | grep collected
Previous HEAD position was 65e58aa4... Ensure coronagraphic associations input/output integrations (#2773)
HEAD is now at c91efdb2... Merge pull request #2730 from nden/doctest-test
collected 909 items / 1 errors

(jwst_dev) rey> git checkout HEAD^ ; pytest --collect-only | grep collected
Previous HEAD position was c91efdb2... Merge pull request #2730 from nden/doctest-test
HEAD is now at e27723ab... Merge pull request #2769 from philhodge/mod_extract_1d
collected 1370 items

(jwst_dev) rey> git checkout HEAD^ ; pytest --collect-only | grep collected
Previous HEAD position was e27723ab... Merge pull request #2769 from philhodge/mod_extract_1d
HEAD is now at e91a8e0d... Merge pull request #2772 from hbushouse/nrs_exptype
collected 1370 items
```

So it seems commit c91efdb24fe47f8bb9ecec3f493c5db2f351f1b9 is the culprit.  @nden not sure what happened there.  I totally missed it.  I would have expected doctest to _add_ tests.

Now to figure out which ones are missing...

**Comment by jdavies-st** [comment]
Perhaps this is the culprit?
https://github.com/spacetelescope/jwst/commit/c91efdb24fe47f8bb9ecec3f493c5db2f351f1b9#diff-380c6a8ebbbce17d55d50ef17d3cf906R26

**Comment by nden** [comment]
Those directories were added to ensure test there are not run. I can track down how many tests each directory removes.

**Comment by nden** [comment]
The majority of the tests are in jwst/lib. As mentioned in [this comment](https://github.com/spacetelescope/jwst/issues/2729#issue-371964374) tests don't run at all because of an error in `jwst/lib/convert_to_uncal.py `. Removing that file runs the rest of the tests in `jwst/lib`: 485. There are 42 tests in `jwst/extern` which I think we don't need to run. `timeconversion` has a different issue, also described in [this comment]( https://github.com/spacetelescope/jwst/issues/2729#issue-371964374) .

**Comment by nden** [comment]
Resolved by #2791
